examples/adsb/src/bin/listen_adsb.rs

```diff
@@ -0,0 +1,142 @@
+use adsb_demod::Decoder;
+use adsb_demod::Demodulator;
+use adsb_demod::PreambleDetector;
+use adsb_demod::Tracker;
+use adsb_demod::DEMOD_SAMPLE_RATE;
+use clap::Parser;
+use futuresdr::anyhow::Result;
+use futuresdr::blocks::Apply;
+use futuresdr::blocks::FileSource;
+use futuresdr::blocks::FirBuilder;
+use futuresdr::blocks::SoapySourceBuilder;
+use futuresdr::blocks::Throttle;
+use futuresdr::log::{warn, LevelFilter};
+use futuresdr::num_complex::Complex32;
+use futuresdr::num_integer;
+use futuresdr::runtime::Flowgraph;
+use futuresdr::runtime::Runtime;
+use std::time::Duration;
+
+#[derive(Parser, Debug)]
+#[clap(version)]
+struct Args {
+    /// Antenna
+    #[clap(short, long)]
+    antenna: Option<String>,
+    /// Soapy Filter
+    #[clap(short, long)]
```

Comment:
Suggest removing `short` from `filter` here as it clashes with `file` later on and causes a panic:

```
thread 'main' panicked at 'Command adsb_demod: Short option names must be unique for each argument, but '-f' is in use by both 'filter' and 'file'',
```

I think it's more UX friendly to expect `-f` to be short for `--file` which is why I propose removing short here, but, more important to get rid of the panic 😬

Reply:
Ah, I did not catch that. Thanks!
At the same time, I updated to use clap 4 syntax.